documentation/en/dev-tools-jaeger-tracing.md

```diff
@@ -1,26 +0,0 @@
-# Jaeger Tracing
```

Comment:
Can't find this on the new site

Reply:
I have not found this very useful as an end user. After a couple of minutes jaeger cannot deal anymore with the amount of traces being emitted. Would it work if I just keep it here in Lotus?

Reply:
Yep, keeping it here works for me

Reply:
added